src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,41 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final DateTimeFormatter DATE_FORMATTER = (
+            DateTimeFormatter.ofPattern("dd.MM.yyyy")
+    );
```

Comment:
```suggestion
    private static final DateTimeFormatter DATE_FORMATTER = 
            DateTimeFormatter.ofPattern("dd.MM.yyyy");
```
redundant brackets

Reply:
Is this a best practice in Java?

Reply:
yes, don't complicate the code unnecessarily